README.md

```diff
@@ -1,23 +1,41 @@
 
+
 # mutex
-    import "github.com/juju/mutex"
+`import "github.com/juju/mutex"`
+
+* [Overview](#pkg-overview)
+* [Index](#pkg-index)
 
+## <a name="pkg-overview">Overview</a>
 package mutex provides a named machine level mutex shareable between processes.
 [![GoDoc](https://godoc.org/github.com/juju/mutex?status.svg)](https://godoc.org/github.com/juju/mutex)
 
-Mutexes have names, and the a mutex with the same name can only be locked by one
-instance of the mutex at a time, even across process boundaries.
+Mutexes have names. Each each name, only one mutex for that name can be
+acquired at the same time, within and across process boundaries. If a
+process dies while the mutex is held, the mutex is automatically released.
+
+The *nix implementation uses flock, while the Windows implementation uses
```

Comment:
This "*" causes The diff Markdown preview to freak out for the entire rest of the file treating *everything* as italics. Hopefully its not actually a problem.

Reply:
I foolishly assumed that godoc2md would escape, but alas. I'm going to be lazy this time and just spell out Linux and MacOS.